safehttp/plugins/htmlinject/htmlinject.go

```diff
@@ -106,6 +121,85 @@ func Transform(src io.Reader, cfg ...Config) (string, error) {
 	return rw.out.String(), nil
 }
 
+// LoadTrustedTemplate processes the given TrustedTemplate with the specified default configurations and
+// adds it to the given template.
+// If the given template is nil a new one is created.
+func LoadTrustedTemplate(tpl *template.Template, csp, xsrf bool, src template.TrustedTemplate) (*template.Template, error) {
```

Comment:
Let's get these all into a separate package, WDYT?

Reply:
What is the value in adding fragmentation to two things that would almost always be used together? Also, note that this package has an extremely small API, I don't think this is bloated. Moreover this package was created to do exactly this job, I am not in favor of creating another to do that job.

Reply:
My feeling was that you aim to create an `htmlinject` package that isn't supposed to work with `safehtml/template` only.

Reply:
Transform is everything you need for html/template.

This patch is just to make it *also* usable with safehtml/template.

Reply:
If you think it would be useful I can duplicate the helpers to also support html/template.

Reply:
I just don't see a good reason to put it here as it seems like a different level of abstraction (i.e. `htmlinject.Transform` and `LoadTrustedTemplate`).

If you have a reason to not do the split (I don't see harm in doing it) then that's okay. LGTM overall as I don't see it as blocking.

Reply:
My reason is just to not bloat our directory structure, and these functions seemed related enough to go in the same package.

We can always do the split in a follow up PR, let's discuss this in person.